src/Okta.Sdk/Generated/ILogGeolocation.Generated.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="ILogGeolocation.Generated.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+// This file was automatically generated. Don't modify it directly.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Okta.Sdk
+{
+    /// <summary>Represents a LogGeolocation resource in the Okta API.</summary>
+    public partial interface ILogGeolocation : IResource
+    {
+        double? Lat { get; }
```

Comment:
Use errata to rename these (only the .NET member names, not the binding names) to `Latitude`, `Longitude`